addons/mail/static/src/models/channel_command/channel_command.js

```diff
@@ -10,7 +10,7 @@ function factory(dependencies) {
 
     ChannelCommand.fields = {
         /**
-         * FIXME use this value task-2343850
+         * The channel types where the command is allowed to be used
```

Comment:
_"Makes only sense for threads on model 'mail.channel'. Type of the channel (e.g. 'chat', 'channel' or 'groups')"_
Would not refer to channel commands in description of this field, at all.